src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-realm-form/rgw-multisite-realm-form.component.ts

```diff
@@ -9,13 +9,14 @@ import { CdValidators } from '~/app/shared/forms/cd-validators';
 import { NotificationService } from '~/app/shared/services/notification.service';
 import { RgwRealm } from '../models/rgw-multisite';
 import { DocService } from '~/app/shared/services/doc.service';
+import { BaseModal } from 'carbon-components-angular';
 
 @Component({
   selector: 'cd-rgw-multisite-realm-form',
   templateUrl: './rgw-multisite-realm-form.component.html',
   styleUrls: ['./rgw-multisite-realm-form.component.scss']
 })
-export class RgwMultisiteRealmFormComponent implements OnInit {
+export class RgwMultisiteRealmFormComponent extends BaseModal implements OnInit {
   action: string;
   multisiteRealmForm: CdFormGroup;
   info: any;
```

Comment:
lets have useful types here.
not part of your PR, if you can do its nice. Not blocking the PR on this.